### DISCUSSION TITLE
Frontend deployment instructions

### DISCUSSION THREAD

**Discussion by nkidambi** [Azure-Samples/graphrag-accelerator#105] [General]
Hi, I have deployed the frontend in Azure as App Service. The instructions in the README file are missing few things. Below are my notes on how to deploy the frontend as App Serivice in Azure. Could it be reviewed and added as needed?  -

## Deploy Frontend to Azure

The folder `frontend` contains the code for the frontend. To deploy this to Azure, follow the steps below
### Pre-requisites
1. Graph Accelerator deployed
2. VS Code
3. Azure Extensions

### Steps
1. Clone the repo and open with VS Code
2. Click on Azure icon 
3. Log into your Azure where you want this deployed.
4. Follow the instructions [here](https://learn.microsoft.com/en-us/answers/questions/1470782/how-to-deploy-a-streamlit-application-on-azure-app) to deploy this streamlit app to Azure
      1. Sample run.sh file
      ```bash
      python -m streamlit run app.py --server.port 8000 --server.address 0.0.0.0
      ``` 

### Configuration settings
Add the following environment variables

| Key | Value | Comments |
| ----- | ---- | ----- |
| DEPLOYMENT_URL | | The APIM **Gateway** url |
| APIM_SUBSCRIPTION_KEY | | The APIM subscription key |

**Comment by Rezanmonis** [comment]
Hello

We are looking for frontend and backend developers as our teammates for our edtech project rezanmonis@gmail.com
